.github/workflows/ci.yml

```diff
@@ -25,7 +26,7 @@ jobs:
       - name: Build contracts
         run: |
           forge --version
-          forge build --sizes
+          forge build
```

Comment:
The build step will always fail because CometFlexVoting is 8kB over the size limit

Reply:
I updated the description of #47 to note that we should also bring back this size check as part of that issue